recipes/flask-restplus/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "flask-restplus" %}
+{% set version = "0.9.2" %}
+{% set sha256 = "c4313097a673ef2cffabceb44b6fdd03132ee5e7ab34d0289c37af12a3d11186" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - flask >=0.8
+    - six >=1.3.0
+    - jsonschema
+    - pytz
+    - aniso8601 >=0.82
```

Comment:
@jakirkham Won't setuptools try to install the dependencies in this case and break with an error that conda doesn't allow to download dependencies?